composer.json

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "spryker-eco/computop-shipment",
+    "type": "library",
+    "description": "ComputopShipment module",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.4",
+        "spryker-eco/computop-extension": "dev-feature/eco-3656/dev-paypal-express as 1.0.0",
+        "spryker/kernel": "^3.30.0",
+        "spryker/shipment": "^7.0.0 || ^8.0.0",
+        "spryker/zed-request": "^3.0.0"
+    },
+    "require-dev": {
+        "phpstan/phpstan": "*",
```

Comment:
Please always try to use concrete versions, or it will fail soon
^0.12 here
etc

Reply:
Do we need to use concrete versions only for phpstan or for all modules in require-dev section?

Reply:
Ideally for all, otherwise also checks like prefer lowest will not be possible